controllers/v1/observationsController.js

```diff
@@ -970,7 +1042,8 @@ module.exports = class Observations extends Abstract {
     * @name importFromFramework
     * @param {Object} req -request Data.
     * @param {String} req.query.frameworkId -framework id.
-    * @param {String} req.query.entityType - entity type name.   
+    * @param {String} req.query.entityType - entity type name. 
+    * @param {String} req.query.programId - program id is required.  
```

Comment:
@aman-tunerlabs - Program id is optional. Also mention program id is external id or internal id